transactions/types.ts

```diff
@@ -0,0 +1,7 @@
+import BigNumber from 'bignumber.js';
+
+export type Brc20Recipient = {
+  address: string;
+  amountBrc20: BigNumber;
+  amountSats: BigNumber;
+};
```

Comment:
I don't think this needs to be in core. It can live in the extension repo since it's not used as an interface with core

Reply:
agreed

Reply:
removed from core and added to extension on latest commits